MAUI/DataGrid/addNew-row.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: post
+title: AddNewRow in .NET MAUI DataGrid control | Syncfusion®
+description: Learn here all about how to add new row in Syncfusion® .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .net maui datagrid, .net maui grid, .net grid maui, .net maui addnewrow, maui add new row
+---
+
+# AddNewRow in MAUI DataGrid (SfDataGrid)
+
+DataGrid provides built-in row (called AddNewRow) that allows user to add new records to underlying collection. Built-in add new row can be enabled or disabled by setting [SfDataGrid.AddNewRowPosition]() property. AddNewRowPosition also denotes the position of add new row in DataGrid.
+
+When you start editing in AddNewRow, the SfDataGrid control creates an instance for the underlying data object and adds it to underlying collection when editing completed.
+
+N>
+1. The underlying data object must be defined with default constructor.
+2. Selection and Editing must be enabled.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       AddNewRowPosition="Top"
+                       SelectionMode="Single"
+                       NavigationMode="Cell"
+                       AllowEditing="True"
+                       AutoGenerateColumnsMode="None"
+                       ItemsSource="{Binding OrderInfoCollection}">
+        <syncfusion:SfDataGrid.Columns>
+            <syncfusion:DataGridNumericColumn HeaderText="Order ID" Format="#" 
+                                            MappingName="OrderID" />
+            <syncfusion:DataGridTextColumn  HeaderText="Customer ID" 
+                                            MappingName="CustomerID.CustomerID" />
+            <syncfusion:DataGridTextColumn  HeaderText="Ship City" 
+                                            MappingName="ShipCity" />
+            <syncfusion:DataGridTextColumn  HeaderText="Ship Country" 
+                                            MappingName="ShipCountry" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataGrid.SelectionMode = DataGridSelectionMode.Single;
+        dataGrid.NavigationMode = DataGridNavigationMode.Cell;
+        dataGrid.AllowEditing = true;
+        dataGrid.AddNewRowPosition = DataGridAddNewRowPosition.Top;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="addnewRowpositon-top" src="Images\addnewrow\maui-dataGrid-AddNewRowPosition-top.png" width="604"/>  
+
+## Changing the AddNewRow Position
+
+AddNewRow position can be changed by setting `SfDataGrid.AddNewRowPosition` property.
```

Comment:
The position of the AddNewRow in `SfDataGrid` can be customized using the `SfDataGrid.AddNewRowPosition` property. By default, this property is set to Top.

The following code snippet demonstrates how to change the AddNewRow position to Bottom in `SfDataGrid`: